src/styles/colour/index.md.njk

```diff
@@ -59,7 +59,7 @@ use `govuk-colour("red")` rather than `$govuk-error-colour`.
 
 ## Colour palette
 
-Use these colours for graphs and supporting material.
+Use these colours for supporting material.
 
 To reference colours from the palette directly you should use the `govuk-colour`
 function. For example, `color: govuk-colour("blue")`.
```

Comment:
Do we have to write 'color' or can we write 'colour'?

Reply:
@EoinShaughnessy That's (unfortunately) the spelling in CSS - it will only recognise `color`, not `colour`. So we'll need to keep this as it is.

Reply:
@vanitabarrett Good to know - thanks!